x-pack/plugins/cloud_security_posture/public/components/accounts_evaluated_widget.tsx

```diff
@@ -7,38 +7,40 @@
 import React from 'react';
 import { EuiFlexGroup, EuiFlexItem, useEuiTheme } from '@elastic/eui';
 import { css } from '@emotion/react';
+import { CLOUD_PROVIDERS, getBenchmarkApplicableTo } from '../../common/utils/helpers';
 import { CIS_AWS, CIS_GCP, CIS_AZURE, CIS_K8S, CIS_EKS } from '../../common/constants';
 import { CISBenchmarkIcon } from './cis_benchmark_icon';
 import { CompactFormattedNumber } from './compact_formatted_number';
 import { useNavigateFindings } from '../common/hooks/use_navigate_findings';
 import { BenchmarkData } from '../../common/types_old';
+import { FINDINGS_GROUPING_OPTIONS } from '../common/constants';
 
 // order in array will determine order of appearance in the dashboard
 const benchmarks = [
   {
     type: CIS_AWS,
-    name: 'Amazon Web Services (AWS)',
-    provider: 'aws',
+    name: getBenchmarkApplicableTo(CIS_AWS),
+    provider: CLOUD_PROVIDERS.AWS,
   },
   {
     type: CIS_GCP,
-    name: 'Google Cloud Platform (GCP)',
-    provider: 'gcp',
+    name: getBenchmarkApplicableTo(CIS_GCP),
+    provider: CLOUD_PROVIDERS.GCP,
   },
   {
     type: CIS_AZURE,
-    name: 'Azure',
-    provider: 'azure',
+    name: getBenchmarkApplicableTo(CIS_AZURE),
+    provider: CLOUD_PROVIDERS.AZURE,
   },
   {
     type: CIS_K8S,
-    name: 'Kubernetes',
-    benchmarkId: 'cis_k8s',
+    name: getBenchmarkApplicableTo(CIS_K8S),
+    benchmarkId: CIS_K8S,
   },
   {
     type: CIS_EKS,
-    name: 'Amazon Elastic Kubernetes Service (EKS)',
-    benchmarkId: 'cis_eks',
+    name: getBenchmarkApplicableTo(CIS_EKS),
+    benchmarkId: CIS_EKS,
```

Comment:
i see that its not new, but can we change it so all of those uses benchmarkId to keep it consistent? not urgent